client/src/main/java/com/paypal/selion/reports/runtime/SaverFileSystem.java

```diff
@@ -131,8 +131,8 @@ public PageContents getScreenshotByName(String name) throws IOException {
     @Override
     public void init() {
         logger.entering();
-        (new File(outputFolder)).mkdirs();
```

Comment:
do we need to handle negative scenario? i.e.. what happens when the outputFolder directory is not created?